config/env/amp.yaml

```diff
@@ -0,0 +1,33 @@
+defaults:
+  - base
+
+_target_: gflownet.envs.amp.AMP
+
+id: amp
+# Minimum and maximum length for the sequences
+min_seq_length: 7
+max_seq_length: 50
+# Number of letters in alphabet
+n_alphabet: 20
+# Minimum and maximum number of steps in the action space
+min_word_len: 1
+max_word_len: 1
+reward_func: power
+reward_norm: 1.0
+reward_norm_std_mult: -1.0
+reward_beta: 8.0
```

Comment:
If the defaults are the same as in base.yaml, then there is no need to add them here too